apps/che/src/main/fabric8/deployment.yml

```diff
@@ -144,13 +144,17 @@ spec:
         imagePullPolicy: "IfNotPresent"
         name: che
         readinessProbe:
-          tcpSocket:
+          httpGet:
+            path: /api/system/state
             port: 8080
-          initialDelaySeconds: 20
+            scheme: HTTP
+          initialDelaySeconds: 60
```

Comment:
@mlabuda @ibuziuk with this config I still get 503 error when che-starter creates a workspace on an idled che-server. I've tried to remove the readiness probe (we need a liveness probe but do we really need a readiness one?) and now I get 504 error.